src/stores/mapStore.ts

```diff
@@ -17,13 +23,18 @@ export class MapStore {
     @observable private _displayCoordinateSystem:CoordinateSystem;
     @observable private _zoom:number;
     @observable private _selectedNodeId: string|null;
+    @observable private _visibleNodeLabels: NodeLabel[];
 
     constructor() {
         this._coordinates = INITIAL_COORDINATES;
         this._zoom = INITIAL_ZOOM;
         this._isMapFullscreen = false;
         this._routes = [];
         this._displayCoordinateSystem = CoordinateSystem.EPSG4326;
+        this._visibleNodeLabels = [
```

Comment:
![screenshot 2018-12-19 at 10 04 38](https://user-images.githubusercontent.com/5954449/50206892-893c5400-0375-11e9-8b47-68510c814b17.png)

I would probably not show both by default, since the default view becomes so messy